README.md

```diff
@@ -98,6 +108,16 @@ Create a new lion instance with the name `simba`
 
 ```ruby
 # code here
+  class Animal
+    attr_accessor :name
+    def intialize(name,greet)
+      @name = name
+      def greet
+        puts @name
+      end
+    end
+    hog = Animal.new(pumba)
```

Comment:
In this case, this should be done outside of the class definition.

Also, `pumba` will result in an error since you don't have a variable with that name. You need to pass in a string instead.